benchexec/tools/symbiotic.py

```diff
@@ -102,12 +104,15 @@ def _getPhase(self, output):
 
         return lastphase
 
-    def determine_result(self, returncode, returnsignal, output, isTimeout):
-        if output is None:
+    def determine_result(self, run):
+        returnsignal = run.exit_code.signal or 0
+        returncode = run.exit_code.value or 0
+
+        if run.output is None:
```

Comment:
Note that the API documentation guarantees that this is wrong. If you want to check for empty output, this needs to be done differently.

Reply:
fixed in b28f883d752cc0656615bceae0bf4a5d0eeedc08